<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerVariableLoggerBrowserWidgetForm</class>
 <widget class="QWidget" name="SeerVariableLoggerBrowserWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="variableAddLineEdit">
       <property name="toolTip">
        <string>Add a variable to the loggger.</string>
       </property>
       <property name="placeholderText">
        <string>Enter a variable name to log</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="variableDeleteToolButton">
       <property name="toolTip">
        <string>Delete selected variables.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</normaloff>:/seer/resources/HighContrast/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="variableDeleteAllToolButton">
       <property name="toolTip">
        <string>Delete all variables.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/edit-delete.svg</normaloff>:/seer/resources/HighContrast/edit-delete.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="variablesTreeWidget">
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Timestamp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
